packages/dev/core/src/Rendering/IBLShadows/iblShadowsVoxelTracingPass.ts

```diff
@@ -2,13 +2,13 @@ import { Constants } from "../../Engines/constants";
 import type { AbstractEngine } from "../../Engines/abstractEngine";
 import type { Scene } from "../../scene";
 import { Matrix, Vector2, Vector4 } from "../../Maths/math.vector";
-import "../../Shaders/iblShadowVoxelTracing.fragment";
 import "../../Shaders/iblShadowDebug.fragment";

```

Comment:
This shader import doesn't need to be here anymore, right?

Reply:
Correct! I'll get that fixed!